pymilo/pymilo_param.py

```diff
@@ -151,6 +157,7 @@
     "SpectralClustering": SpectralClustering,
     "AgglomerativeClustering": AgglomerativeClustering,
     "DBSCAN": DBSCAN,
+    "HDBSCAN": NOT_SUPPORTED if not hdbscan_support else HDBSCAN,
```

Comment:
why not:
`"HDBSCAN": HDBSCAN if hdbscan_support else NOT_SUPPORTED`

Reply:
Got you, yes this one is easier to read.
@sadrasabouri